app/lib/webpack/create-chain.js

```diff
@@ -322,7 +322,7 @@ module.exports = function (cfg, configName) {
           ignore: [ appPaths.resolve.app('/**/.*') ].concat(
             // avoid useless files to be copied
             ['electron', 'cordova', 'capacitor'].includes(cfg.ctx.modeName)
-              ? [ appPaths.resolve.app('public/icons'), appPaths.resolve.app('public/favicon.ico') ]
+              ? [ '**/icons/**/*', '**/favicon.ico' ]
```

Comment:
This can be simplified. Also, even though the chances are low, deep matching the icons directory could result in unwanted results.
```suggestion
              ? [ 'icons/**/*', 'favicon.ico' ]
```

Reply:
That doesn't work. How I have done it is the suggested way I've seen online.

Reply:
See: https://github.com/mrmlnc/fast-glob#ignore

Reply:
For anyone else who reads: after discussion in discord, it needs to be as it is now. Glob options are strange with copy-webpack plugin.

Reply:
For info:

https://github.com/webpack-contrib/copy-webpack-plugin/blob/21d9c7d3217c472bc195ec7461a3b26e1360c886/src/utils/createPatternGlob.js#L30-L33

it appends **/*
so it's like there are multiple inputs
so we can't just ignore folder/*
but **/folder/* works

So, if we use glob matching in from instead, we could make it relative.
But I don't think it's required at this point. The fix is inline with docs and works